### DISCUSSION TITLE
New Releases Subscription

### DISCUSSION THREAD

**Discussion by joshmedeski** [neovim/neovim#24205] [General]
When I subscribe to Neovim releases via GitHub I get a notification for every nightly release (which doesn't matter to me that much). I'd love a way to subscribe to only new tagged version releases (ex: [v0.9.1](https://github.com/neovim/neovim/releases/tag/v0.9.1)).

I also noticed the website news isn't up-to-date with the latest releases (the last post is for `v0.7`, but `v0.9.1` is the latest release).

Perhaps we could automate all GitHub tagged releases that aren't `nightly` to auto-publish a news entry on the website?

**Comment by clason** [comment]
The release newslettersare hand-written and nothing that can be automated -- which is why there haven't been any for the recent releases. 

We have added a `:h news.txt` entry to make up for it.

**Comment by clason** [comment]
Regarding the subscription: that is something you have to request from GitHub; the only thing we could do (but would not consider, for obvious reasons) is to stop releasing nightlies.

**Comment by xbc5** [comment]
> that is something you have to request from GitHub

Where would I request that?
